src/Runner.Worker/StepsRunner.cs

```diff
@@ -246,6 +247,13 @@ public async Task RunAsync(IExecutionContext jobContext)
 
                 Trace.Info($"Current state: job state = '{jobContext.Result}'");
             }
+
+            if (jobContext.JobContext.ContainsKey("Node12ActionsWarnings"))
```

Comment:
@thboop let me know what you think of creating an `IActionCommandManager` here and printing the warning.

It's outside the main `while` loop and I wonder if it can have unforeseen consequences.

Otherwise it's fairly safe, the `if` body will only execute if the FF is on because `"Node12ActionsWarnings"` is only set if the FF is on.